pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -179,42 +179,46 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 	)
 
 	Context("tolerations with eviction policies given", func() {
-		var vmi *v1.VirtualMachineInstance
-		var policyMigrate = v1.EvictionStrategyLiveMigrate
-		var policyMigrateIfPossible = v1.EvictionStrategyLiveMigrateIfPossible
-		var policyNone = v1.EvictionStrategyNone
-		var policyExternal = v1.EvictionStrategyExternal
-
 		BeforeEach(func() {
 			enableFeatureGate(deprecation.LiveMigrationGate)
```

Comment:
The live migration feature gate was [GAed](https://github.com/kubevirt/kubevirt/pull/8047) a long time ago and should not affect the tested logic.
I think it should be safe to remove it and thus drop the whole `BeforeEach` stanza.

Reply:
Good idea. Done.